ci/common

```diff
@@ -94,8 +99,10 @@ await_dns() {
 }
 
 # This function can be used to ensure we can resolve jupyter.org as an external
-# domain name from a pod in the K8s cluster, as required by netpol tests.
-await_dns_with_hub_pod() {
+# domain name from a pod in the K8s cluster, as required by netpol tests. It is
+# not currently used in .travis.yml but is useful if we ever want to install
+# canal instead of calico for Travis CI. This is how it failed.
+await_dns_functionality_using_hub_pod() {
```

Comment:
Not a blocker, but what do you think about removing unused code to avoid confusing people- it's in the git commit history anyway? Alternatively does this function succeed with calico- if so how about include it in travis anyway? It'll provide advance notice of (possibly external) DNS or network failures, and might avoid some unnecessary debugging of failures.

Reply:
I've considered this specifically, I also don't like to clutter it, but this will only be useful for me and if it resides in the git history as we are the only ones aware of it.

DNS failures of varying kind are detected as part of the netpol test atm. I run a specific check to verify DNS functionality before allow/deny rules are tested because it took quite a while for me to figure out these kinds of errors :p

I'll go ahead and remove it!